src/main/java/io/jenkins/plugins/ml/KernelInterpreter.java

```diff
@@ -0,0 +1,12 @@
+package io.jenkins.plugins.ml;
```

Comment:
Are we including any copyright header to the source code? I think the licenses state it's not necessary but may be useful for users.

Reply:
I will add the copyright header. But not sure, about whom to be mentioned in the text.